tilelang/language/dtypes.py

```
@@ -182,6 +183,11 @@ def __dtype_call__(self: dtype, *args, is_size_var: bool = False) -> tirx.Var:
     return call(expr, is_size_var)
 
 
+@functools.lru_cache(maxsize=None)
```

Ruff (0.16.3)

[warning] Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)